pkgs/development/compilers/miranda/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl, gcc }:
+
+stdenv.mkDerivation rec {
+  version = "2.066";
+  pname = "miranda";
+
+  src = fetchurl {
+    url = "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-${builtins.replaceStrings [ "." ] [ "" ] version}-src.tgz";
+    sha256 = "1sqn031agwwgq3pka55pxd6m22dagmkxjhbwrni8gpigihg2h72j";
+  };
+
+  enableParallelBuilding = true;
+
+  configurePhase = ''
```

Comment:
Also when overriding `configurePhase`, you should also provide `runHook preConfigure` and `runHook postConfigure`. Though, shebang patching is conventionally done in `postPatch`.